src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -119,10 +140,11 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
             this.props.selectedAsset.asset.size.width,
             this.props.selectedAsset.asset.size.height,
         );
+        const lockedTags = this.props.lockedTags;
         const newRegion = {
             id,
             type: this.editorModeToType(this.props.editorMode),
-            tags: [],
+            tags: (lockedTags) ? lockedTags : [],
```

Comment:
you can simplify this as `lockedTags || []` If `lockedTags` is defined it will use its value, otherwise an empty array.

Reply:
Good catch